src/auro-flight-header.js

```diff
@@ -69,13 +69,34 @@ class AuroFlightHeader extends LitElement {
       : html``;
   }
 
+  readFlight(flight) {
```

Comment:
@aesthetiques Why abstract this logic out to its own function? Why not just do the following on link 84??

`Array.from(this.flightType()).join(' ')`

Reply:
This is really the difference between functional and procedural code. In this context, I can go either way, but if this is to be a function, there need to be JSDoc comments added.

Reply:
In-lined